apps/coursition/src/app/api/payments/route.ts

```diff
@@ -25,6 +25,7 @@ export const POST = async (request: NextRequest) => {
           },
         })
         revalidatePath('/')
+        return { success: true, message: 'user.roles.upated' }
```

Comment:
**question:** Message is only for us, right? Because Lemon Squeezy doesn't work with it in any way, right? If so, I think we should go more formal approach like we do in API routes. Let's remove this message part and add it in follow up PR.